packages-ts/starknet-gauntlet-ocr2/test/commands/ocr2.test.ts

```diff
@@ -182,14 +181,15 @@ describe('OCR2 Contract', () => {
       // TODO: use StarknetContract decodeEvents from starknet-hardhat-plugin instead
       const eventData = receipt.events[0].data
       // reconstruct signers array from event
-      let eventSigners = []
+      let eventSigners: bigint[] = []
       for (let i = 0; i < signers.length; i++) {
-        const signer = new BN(eventData[4 + 2 * i].replace('0x', ''), 16) // split according to event structure
+        const signer = BigInt(eventData[4 + 2 * i]) // split according to event structure
         eventSigners.push(signer)
       }
 
       expect(eventSigners).toEqual(
-        signers.map((s) => new BN(s.replace('ocr2on_starknet_', '').replace('0x', ''), 16)),
+        // eaiser to remove prefix and 0x and then add 0x back
+        signers.map((s) => BigInt(`0x${s.replace('ocr2on_starknet_', '').replace('0x', '')}`)),
```

Comment:
Do we need this extra replace?

```suggestion
        signers.map((s) => BigInt(`s.replace('ocr2on_starknet_', '')`)),
```

Reply:
Yeah, some of the testcases have the "0x" prefix and some don't. So it's easier to say "let's remove 0x if you have it and add it back later" rather than "let's only add 0x if you don't have it". 

Same thing for the ocr2_on_starknet_ infix as well